spec.md

```diff
@@ -151,6 +151,21 @@ Service Broker MAY reject the request with `412 Precondition Failed` and
 provide a message that informs the operator of the API version that is to be
 used instead.
 
+## Extending Resources
+
+Senders of messages defined by this specification MAY include additional
+fields within the JSON objects as extensions. For backwards compatibility
+reasons, new fields MAY be added at any location within the JSON objects,
+however, they SHOULD be placed within the `extensions` property of the JSON
+object.
+
+When adding new fields, unique prefixes SHOULD be used for the field names to
```

Comment:
I would strike this, if the OEM is putting prefixes, they might as well put the data inside the extensions location.